src/notifications/notificationmanager.cpp

```diff
@@ -545,22 +544,15 @@ uint NotificationManager::handleNotify(int clientPid, const QString &appName, ui
 
     notification->restartProgressTimer();
 
-    const QString previewSummary(hints_.value(LipstickNotification::HINT_PREVIEW_SUMMARY).toString());
-    const QString previewBody(hints_.value(LipstickNotification::HINT_PREVIEW_BODY).toString());
-
+    // Use the summary and body as fallback values for previewSummary and previewBody.
+    // (This can be avoided by setting them explicitly to empty strings.)
+    if (!hints_.contains(LipstickNotification::HINT_PREVIEW_SUMMARY)) {
+        hints_.insert(LipstickNotification::HINT_PREVIEW_SUMMARY, summary);
+    }
+    if (!hints_.contains(LipstickNotification::HINT_PREVIEW_BODY)) {
+        hints_.insert(LipstickNotification::HINT_PREVIEW_BODY, body);
+    }
```

Comment:
Wonder if we need these or do they make any difference as neno-notifications / Notification::publish() also has similar fallback mechanism.

Reply:
Yeah I'll drop that